### DISCUSSION TITLE
Quasar + testing-library integration

### DISCUSSION THREAD

**Discussion by rldorado** [quasarframework/quasar#9238] [General - Components / Directives / etc]
Hi there, first time writing here so my apologies if I miss something :)

I'm writing on relation to this topic on the forum: 
https://forum.quasar-framework.org/topic/4835/integration-with-testing-library

On my case I'm using `vue-cli` with `vue-cli-plugin-quasar` and I'm trying to do tests with [testing-library](https://github.com/testing-library) but unfortunately I don't know how should I require Quasar to work as I always receive:

```
[Vue warn]: Unknown custom element: <q-page> - did you register the component correctly? For recursive components, make sure to provide the "name" option.
```

I've been taking a look on a Vuetify example within their own repo but I'm not able to replicate a custom container example for Quasar:
https://github.com/testing-library/vue-testing-library/blob/master/src/__tests__/vuetify.js

Any ideas on how should I proceed? I feel a bit lost, thank you in advance.

**Comment by sujandev** [comment]
@rldorado  any update on this issue ?  i am getting same problem .  v2 is ok but problem is with v1
